app/controllers/api/abstract_controller.rb

```diff
@@ -21,7 +21,8 @@ class AbstractController < ApplicationController
 
     def authenticate_user!
       return true if current_user
-      auth = Auth::Create.run(request.headers.to_h)
+      auth = Auth::Create.run(bot_token: request.headers["HTTP_BOT_TOKEN"],
+                              bot_uuid:  request.headers["HTTP_BOT_UUID"])
```

Comment:
Unnecessary spacing detected.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.